src/pages/_document.tsx

```diff
@@ -38,6 +38,12 @@ export default class MyDocument extends Document {
             href="/icons/safari-pinned-tab.svg"
             color="#caeaf8"
           />
+          <link rel="manifest" href="/manifest.json" />
+          <meta name="theme-color" content="#fefefe" />
```

Comment:
Not 100% sure we need this as it's already specified in public/manifest.json should probably double check this

Reply:
We do need the tag (see: https://github.com/feelingapp/feeling-app/runs/1168465935?check_suite_focus=true) - I've changed it to match the one in manifest